integration_tests/test_client_integrated.py

```diff
@@ -242,3 +242,51 @@ def test_transformSectionClient(render, teststack,
     assert all([ts.tforms[-1].to_dict() == tform.to_dict()
                 for ts in output_ts])
     renderapi.stack.delete_stack(deststack, render=render)
+
+
+def validate_mipmap(root_directory, tilespec, minlevel, maxlevel, extension):
+    parts = urllib.parse.urlparse(tilespec.ip.get(0)['imageUrl'])
```

Comment:
I think urllib.parse is not 2.7 compatible (unless I'm missing something) and this should also use unquote in case of, e.g., spaces in the filename. The formula I've been using is `urllib.unquote(urlparse.urlparse('file:///path/to/my%20file.png').path)`, which I think does everything we need.